shared/proc/CellClassification/waveWidths.m

```diff
@@ -0,0 +1,114 @@
+function [pw,vw,asy,v2p] = waveWidths(mWV)
```

Comment:
this could be a good function to write some tests for! open an issue with some ideas?